opentracing-jfr-tracer/src/main/java/io/opentracing/contrib/jfrtracer/impl/wrapper/SpanContextUtil.java

```diff
@@ -0,0 +1,49 @@
+package io.opentracing.contrib.jfrtracer.impl.wrapper;
+
+import brave.opentracing.BraveSpanContext;
+import io.jaegertracing.internal.JaegerSpanContext;
+import io.opentracing.SpanContext;
+
+/**
+ * SpanContextUtil allows to extract traceId and spanId from tracers current implementation should
+ * be simplified when OpenTracing API for Java version 0.32.0 is released
+ */
+//TODO: simplify when Jaeger and Brave supports 0.32.0
+public final class SpanContextUtil {
+
+	/**
+	 * extract traceId from SpanContext
+	 * 
+	 * @param context
+	 *            SpanContext
+	 * @return traceId
+	 */
+	public static String getTraceIdBySpanContext(SpanContext context) {
+		if (context instanceof JaegerSpanContext) {
```

Comment:
I just removed my code that added "ContextExtractors". That system was extensible, but also introduced third-party dependencies what we don't want. This is a less useful (hardcoded) variant of the ContextExtractors.